### PR TITLE
Expand tabs

### PR DESCRIPTION
**Pull request by asmacdo**
Just fixing a minor annoyance :)

AFAIK you'd always want expandtab for python.

### CODE REVIEW

File: datalad_metalad/tests/test_filter.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
 # -*- coding: utf-8 -*-
-# ex: set sts=4 ts=4 sw=4 noet:
+# ex: set sts=4 ts=4 sw=4:
```
Comment (yarikoptic) [inline datalad_metalad/tests/test_filter.py:3]:
```suggestion
# ex: set sts=4 ts=4 sw=4 et:
```

**Review by christian-monch — APPROVED** [review]
Thx a lot @asmacdo

File: datalad_metalad/tests/test_filter.py (hunk 1 above)
Comment (asmacdo) [reply datalad_metalad/tests/test_filter.py]: did it :)

**Comment by yarikoptic** [comment]
Looks good @christian-monch  -- let's merge?

**Pull request merged**